autoload/lsp/utils.vim

```diff
@@ -82,16 +82,22 @@ if has('win32') || has('win64')
             let s:path_to_uri_cache[a:path] = a:path
             return s:path_to_uri_cache[a:path]
         else
+            " Transform cygwin paths to windows paths
+            let l:path = a:path
+            if has('win32unix')
+                let l:path = substitute(a:path, '\c^/\([a-z]\)', '\U\1:', '')
```

Comment:
This break `substitute('/path/to/dir', '\c^/\([a-z]\)', '\U\1:', '')`.

Reply:
Probably this is correct.
```
let l:path = substitute(l:path, '\c^/\([a-z]\)/', '\U\1:/', '')
```

Reply:
Yes you are correct. Resolved